modules/nf-core/cellbender/merge/templates/merge.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import platform
+import cellbender
+import anndata as ad
+from cellbender.remove_background.downstream import load_anndata_from_input_and_output
+
+def format_yaml_like(data: dict, indent: int = 0) -> str:
+    """Formats a dictionary to a YAML-like string.
+
+    Args:
+        data (dict): The dictionary to format.
+        indent (int): The current indentation level.
+
+    Returns:
+        str: A string formatted as YAML.
+    """
+    yaml_str = ""
+    for key, value in data.items():
+        spaces = "  " * indent
+        if isinstance(value, dict):
+            yaml_str += f"{spaces}{key}:\\n{format_yaml_like(value, indent + 1)}"
+        else:
+            yaml_str += f"{spaces}{key}: {value}\\n"
+    return yaml_str
```

Comment:
Really no pyyaml in the cellbender container? Either way not that important, feel free to leave as is.